CMakeLists.txt

```diff
@@ -616,6 +616,26 @@ if (WITH_OPENH264 AND NOT OPENH264_FOUND)
 endif()
 set(WITH_OPENH264 ${OPENH264_FOUND})
 
+option(WITH_MS_RDPECAM_CLIENT "Build with video camera redirect client" OFF)
+if(WITH_MS_RDPECAM_CLIENT)
+	# currently camera redirect client supported for platforms with Video4Linux only
+	find_package(V4L)
+	if(V4L_FOUND)
+		message(STATUS "libv4l-dev found")
+		option(WITH_V4L "Use libv4l-dev for camera redirect" ${V4L_FOUND})
+	else()
+		message(FATAL_ERROR "libv4l-dev required for WITH_MS_RDPECAM_CLIENT")
+	endif()
+
+	if(NOT WITH_FFMPEG AND NOT WITH_SWSCALE)
+		message(FATAL_ERROR "WITH_SWSCALE required for WITH_MS_RDPECAM_CLIENT")
+	endif()
+
+	if(NOT WITH_OPENH264 AND NOT WITH_FFMPEG)
+		message(FATAL_ERROR "WITH_OPENH264 or WITH_FFMPEG required for WITH_MS_RDPECAM_CLIENT")
```

Comment:
you require `WITH_VIDEO_FFMPEG` or `h264_context_new` will fail

Reply:
This logic is revised and moved to `channels/rdpecam/client/CMakeLists.txt`, as suggested.
`swscale` is a required library and it is a part of `FFMPEG`, but it also exists as standalone library and can be used without `FFMPEG` (i.e. with `OPENH264`).